pages/chat/[[...roomId]].js

```diff
@@ -169,30 +186,112 @@ export default function Chat() {
         .sortBy(sortRooms)
         .value();
 
+    const toggleRoomListView = () => {
+        setIsRoomListVisible(prevState => {
+            if (!iframe.current) setIsLoadingIframe(true);
+            else setIsLoadingIframe(false);
+
+            return !prevState;
+        });
+        router.push('/chat');
+    };
+
+    const handleLeave = async (roomId) => {
+        setIsLeavingRoom(true);
+        await matrix.leaveRoom(roomId)
+            .catch(error => console.debug(error));
+        setIsLeavingRoom(false);
+    };
+
+    useEffect(() => {
+        if (roomId) {
+            if (!iframe.current) {
+                setIsLoadingIframe(true);
+            }
+
+            setIsRoomListVisible(false);
+        } else {
+            setIsLoadingIframe(false);
+        }
+    }, [roomId]);
+
+    useEffect(() => {
+        function handleResize() {
+            setWindowWidth(window.innerWidth);
+        }
+
+        handleResize();
+        window.addEventListener('resize', handleResize);
+
+        return () => {
+            window.removeEventListener('load', handleResize);
+            window.removeEventListener('resize', handleResize);
+        };
+    }, []);
+
     return (
         <>
-            <DefaultLayout.Sidebar>
-                <h2>/chat</h2>
+            <Sidebar roomId={!!roomId} isRoomListVisible={isRoomListVisible}>
+                <ServiceSubmenu
+                    title={<h2>/chat</h2>}
+                    onClick={toggleRoomListView}
+                    icon={<Icon><ChatNewIcon /></Icon>}
+                />
                 <details open>
-                    <summary><h3 style={{ display: 'inline-block', marginBottom: '1rem' }}>{ t('People') }</h3></summary>
-                    { directMessages && directMessages.map((room) => <SidebarListEntry key={room.roomId} room={room} />) }
+                    <Summary><h3>{ t('People') }</h3></Summary>
+                    <ServiceTable>
+                        <ServiceTable.Body>
+                            { directMessages && directMessages.map((room) => <SidebarListEntry key={room.roomId} room={room} selected={room.roomId === roomId} />) }
+                        </ServiceTable.Body>
+                    </ServiceTable>
                 </details>
                 <br />
                 <details open>
-                    <summary><h3 style={{ display: 'inline-block', marginBottom: '1rem' }}>{ t('Rooms') }</h3></summary>
-                    { otherRooms && otherRooms.map((room) => <SidebarListEntry key={room.roomId} room={room} />) }
+                    <Summary><h3>{ t('Rooms') }</h3></Summary>
+                    <ServiceTable>
+                        <ServiceTable.Body>
+                            { otherRooms && otherRooms.map((room) => {
+                                return <ServiceTable key={room.roomId}><SidebarListEntry key={room.roomId} room={room} selected={room.roomId === roomId} /></ServiceTable>;
```

Comment:
🤔 

A `<ServiceTable>` inside of a `<ServiceTable>` (line 251)?
This results in this double-border again:

<img width="376" alt="image" src="https://github.com/medienhaus/medienhaus-spaces/assets/706419/719bf520-1059-4454-8274-30066ce356f9">